docs/components/block-preview.tsx

```
@@ -0,0 +1,193 @@
+          label="Tablet"
+        >
+          <svg
+            width="16"
+            height="16"
+            viewBox="0 0 24 24"
+            fill="none"
+            stroke="currentColor"
+            strokeWidth="2"
+            strokeLinecap="round"
+            strokeLinejoin="round"
+          >
```

Biome (2.4.7)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)